public/userInfo.php

```diff
@@ -108,14 +108,17 @@
 
 $pageTitle = "$userPage";
 
-$userPagePoints = getPlayerPoints($userPage);
+$userPagePoints = 0;
+if (getPlayerPoints($userPage, $userPoints)) {
+    $userPagePoints = $userPoints['RAPoints'] + $userPoints['RASoftcorePoints'];
+}
```

Comment:
`$userPagePoints` doesn't appear to be used anywhere. I believe you can eliminate this call.

Reply:
Hmm yeah you are right this one wasn't being used at all